src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -25,12 +26,14 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
+    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
```

Comment:
If the goal is to not sit at 2 charges then you should keep it on cooldown and get pretty close to max estimated CPM minus 1 cast, so this seems a bit low.